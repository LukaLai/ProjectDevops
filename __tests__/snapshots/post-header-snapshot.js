import { render } from '@testing-library/react'
import { PostHeader } from '../../src/app/_components/post-header'

it('renders homepage unchanged', () => {
    const { container } = render(<PostHeader date={'2024-04-17T05:35:07.322Z'} author={'jj'} title={'test'} coverImage={'/public/assets/blog/authors/jj.jpeg'}/>)
    expect(container).toMatchSnapshot()
})