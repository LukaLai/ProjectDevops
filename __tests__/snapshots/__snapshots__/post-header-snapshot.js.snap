// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <h1
    class="text-5xl md:text-7xl lg:text-8xl font-bold tracking-tighter leading-tight md:leading-none mb-12 text-center md:text-left"
  >
    test
  </h1>
  <div
    class="hidden md:block md:mb-12"
  >
    <div
      class="flex items-center"
    >
      <img
        alt="jj"
        class="w-12 h-12 rounded-full mr-4"
        data-nimg="1"
        decoding="async"
        height="12"
        loading="lazy"
        src="/_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=32&q=75"
        srcset="/_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=16&q=75 1x, /_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=32&q=75 2x"
        style="color: transparent;"
        width="12"
      />
      <div
        class="text-xl font-bold"
      >
        jj
      </div>
    </div>
  </div>
  <div
    class="mb-8 md:mb-16 sm:mx-0"
  >
    <div
      class="sm:mx-0"
    >
      <img
        alt="Cover Image for test"
        class="shadow-sm w-full"
        data-nimg="1"
        decoding="async"
        height="630"
        loading="lazy"
        src="/_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=3840&q=75"
        srcset="/_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=1920&q=75 1x, /_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=3840&q=75 2x"
        style="color: transparent;"
        width="1300"
      />
    </div>
  </div>
  <div
    class="max-w-2xl mx-auto"
  >
    <div
      class="block md:hidden mb-6"
    >
      <div
        class="flex items-center"
      >
        <img
          alt="jj"
          class="w-12 h-12 rounded-full mr-4"
          data-nimg="1"
          decoding="async"
          height="12"
          loading="lazy"
          src="/_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=32&q=75"
          srcset="/_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=16&q=75 1x, /_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=32&q=75 2x"
          style="color: transparent;"
          width="12"
        />
        <div
          class="text-xl font-bold"
        >
          jj
        </div>
      </div>
    </div>
    <div
      class="mb-6 text-lg"
    >
      <time
        datetime="2024-04-17T05:35:07.322Z"
      >
        April	17, 2024
      </time>
    </div>
  </div>
</div>
`;
